ParisBNF/abb/BNFabb115.xml

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb115" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Prayer of Moses, The Wisest among the Wisest Ones</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt>
+                <p/>
+            </editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas:  Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                          <idno>BnF Éthiopien d'Abbadie 115</idno>
+                  </msIdentifier>
+
+                    <msContents>
+                      <msItem xml:id="ms_i1">
+                        <locus from="2r" to="49r"/>
+                         <title type="complete" ref="LIT4002Salotz"></title>
+                         <textLang mainLang="gez"/>
+                         <incipit xml:lang="gez">
+                           ንነግረክሙ፡ አኃውየ፡ መግረሬ፡ ፀር፡ ወመድኅነ፡ ሙሴ፡ እምእደ፡ ፈርዖን፡ ዘወሀቦ፡ ሙሴ፨
+                           </incipit>
+                           <note>The prayer is divided into four parts that
+                           begin on <locus target="#2r"/>, <locus target="#28v"/>,
+                         <locus target="#31r"/>, <locus target="#44v"/>, resp. The name of the scribe, <persName ref="PRS13744Giyorgis">Giyorgis</persName>,
+                         appears in the concluding supplication on <locus target="#49r"/>. C. Conti Rossini suggests
+                       that this specific version of the text may be of <persName ref="ETH1274Betaes"/> origin.</note>
```

Comment:
I assume that since the text of the manuscript is not available to us, it's not possible to confirm and hasn't been studied yet? Otherwise it would make sense to add this information to the work record (even if it remains a general record)

Reply:
I agree, though this is an assumption only (A. d'Abbadie used to indicate Beta Esrael origin of the manuscripts, in this case he did nothing). There is an issue of the miniatures of in this manuscripts, which makes the Beta Esrael origin not very likely.